src/Symfony/Component/PropertyInfo/PropertyInfo.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * Default {@see PropertyInfoInterface} implementation.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PropertyInfo implements PropertyInfoInterface
+{
+    /**
+     * @var PropertyListRetrieverInterface[]
+     */
+    private $listExtractors;
```

Comment:
would add white spaces between the var and the next doc

same for the next member vars in this class